### PR TITLE
Minor ruby fixes

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

**Comment by juancarlospaco** [comment]
Looks nice.

File: rest/accounts/list-get-example-2/list-get-example-2.4.x.rb [hunk 1]
```diff
@@ -10,6 +10,6 @@
 
 # Loop over accounts and print out a property for each one
 @client.accounts.list({
-    :status => "active"}).each do |account|
-    puts account.friendly_name
+  :status => "active"}).each do |account|
```
Comment (acamino) [inline rest/accounts/list-get-example-2/list-get-example-2.4.x.rb]: No need for curly braces in this example. Please take a look at https://github.com/bbatsov/ruby-style-guide#no-braces-opts-hash

File: client/capability-token-2way/capability-token.4.x.rb [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
-    capability = Twilio::Util::Capability.new account_sid, auth_token
-    # Create an application sid at 
-    # twilio.com/console/phone-numbers/dev-tools/twiml-apps and use it here
-    capability.allow_client_outgoing "APXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
-    capability.allow_client_incoming params['ClientName']
-    token = capability.generate
+  account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
+  auth_token = 'your_auth_token'
+  capability = Twilio::Util::Capability.new account_sid, auth_token
+  # Create an application sid at
```
Comment (acamino) [inline client/capability-token-2way/capability-token.4.x.rb]: What about capitalizing the acronym SID? I noticed you already did it with other examples.

File: rest/addresses/list-get-example-2/list-get-example-2.4.x.rb [hunk 3]
```diff
@@ -10,6 +10,6 @@
 
 # Loop over addresses and print out a property for each one
 @client.account.addresses.list({
-    customer_name: "Customer 123"}).each do |address|
-    puts address.friendly_name
+  customer_name: "Customer 123"}).each do |address|
```
Comment (acamino) [inline rest/addresses/list-get-example-2/list-get-example-2.4.x.rb]: ditto - about using curly braces when you're passing a hash to a method.

Comment (acamino) [inline rest/addresses/list-get-example-2/list-get-example-2.4.x.rb]: I'd prefer using a one-liner, like this:
```ruby
@client.account.addresses.list(customer_name: "Customer 123").each do |address|
  # ...
```

File: rest/making-calls-sip/example-1/example-1.4.x.rb [hunk 4]
```diff
@@ -8,6 +8,6 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 call = @client.account.calls.create(:url => "http://www.example.com/sipdial.xml",
```
Comment (acamino) [inline rest/making-calls-sip/example-1/example-1.4.x.rb]: In an example like this you should align the rest of the params considering the position of `:url`.

```ruby
call = @client.account.calls.create(:url => "http://www.example.com/sipdial.xml",
                                    :to => "sip:kate@example.com",
                                    :from => "Jack")

puts call.direction
```

Please take a look at https://github.com/bbatsov/ruby-style-guide#no-double-indent

File: rest/making-calls-sip/example-2/example-2.4.x.rb [hunk 5]
```diff
@@ -8,8 +8,8 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 call = @client.account.calls.create(:url => "http://www.example.com/sipdial.xml",
-    :to => "sip:kate@example.com",
-    :from => "Jack",
-    :sip_auth_password => "secret",
-    :sip_auth_username => "jack")
+  :to => "sip:kate@example.com",
```
Comment (acamino) [inline rest/making-calls-sip/example-2/example-2.4.x.rb]: ditto - Seems like you should apply the same changes to all the examples that has multiple params.

File: rest/sip-in/delete-credential/delete-credential.4.x.rb [hunk 6]
```diff
@@ -8,4 +8,4 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 @credential = @client.account.sip.credential_lists.get("CL32a3c49700934481addd5ce1659f04d2").credentials.get("SC32a3c49700934481addd5ce1659f04d2")
```
Comment (acamino) [inline rest/sip-in/delete-credential/delete-credential.4.x.rb]: This line is too long. What do you think about splitting it into several lines? - you can find more info about the recommended practices at https://github.com/bbatsov/ruby-style-guide#consistent-multi-line-chains. I found **Option A** more intuitive.

```ruby
@credential = @client.account.sip
  .credential_lists
  .get("CL32a3c49700934481addd5ce1659f04d2")
  .credentials
  .get("SC32a3c49700934481addd5ce1659f04d2")
```

File: rest/subaccounts/listing-subaccounts-example-2/listing-subaccounts-example-2.4.x.rb [hunk 7]
```diff
@@ -9,5 +9,5 @@
 
 # Loop over accounts and print out a property for each one
 @client.accounts.list({:friendly_name => "MySubaccount"}).each do |account|
```
Comment (acamino) [inline rest/subaccounts/listing-subaccounts-example-2/listing-subaccounts-example-2.4.x.rb]: ditto - about curly braces usage.

File: rest/taskrouter/workflows/list/post/example-1/example-1.4.x.rb [hunk 8]
```diff
@@ -33,4 +33,4 @@
 json = config.to_json
 
 workflow = client.workspace.workflows.create(friendly_name: 'Sales, Marketing, Support Workflow', assignment_callback_url: 'http://example.com', fallback_assignment_callback_url: 'http://example.2com', task_reservation_timeout: '30', configuration: json)
```
Comment (acamino) [inline rest/taskrouter/workflows/list/post/example-1/example-1.4.x.rb]: This one could be broken into several lines. I know the original author put this example like this, but we can make it more readable. 😉

File: rest/test-credentials/test-calls-example-1/test-calls-example-1.4.x.rb [hunk 9]
```diff
@@ -8,6 +8,6 @@
 @client = Twilio::REST::Client.new account_sid, auth_token
 
 call = @client.account.calls.create(:url => "http://demo.twilio.com/docs/voice.xml",
-    :to => "+14108675309",
-    :from => "+15005550006")
+  :to => "+14108675309",
```
Comment (acamino) [inline rest/test-credentials/test-calls-example-1/test-calls-example-1.4.x.rb]: ditto - about alignment.

**Comment by acamino** [comment]
Thanks @hortega. I'm closing this one in favor of #211 and #212.

**Pull request closed without merging**